### PR TITLE
Fix beehive alignment

### PR DESCRIPTION
**Pull request by reyqn**
## Description

Beehive alignment is broken because when giving latitude / longitude to pgoapi, it returns the latitude and longitude of a place next to what we want. So it isn't very accurate.
I added a way to give exact latitude and longitude as a location. This should fix the issue.

## Motivation and Context

Beehive scan isn't optimal. Some people were complaining about it so here is a fix.

## How Has This Been Tested?

I tested it quickly so feel free to check it a little more. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: runserver.py [hunk 1]
```diff
@@ -55,10 +55,14 @@
         logging.getLogger('pgoapi').setLevel(logging.DEBUG)
         logging.getLogger('rpc_api').setLevel(logging.DEBUG)
 
-    position = get_pos_by_name(args.location)
-    if not any(position):
-        log.error('Could not get a position by name, aborting')
-        sys.exit()
+
+    if  (args.latitude is not None and args.longitude is not None):
```
Comment (Ashex) [inline runserver.py]: Is there any reason you can't just do a validation check here to determine if `--location` is a GPS coordinate and handle it here?

Adding extra arguments when location is available does make this a little more difficult to maintain.

Comment (reyqn) [inline runserver.py]: Cause I was lazy

**Comment by BaIthamel** [comment]
What is wrong with beehive?

**Comment by reyqn** [comment]
@BaIthamel https://github.com/AHAAAAAAA/PokemonGo-Map/pull/3106#issuecomment-236888039

**Comment by BaIthamel** [comment]
I fixed that already #2815

**Comment by reyqn** [comment]
@BaIthamel well yes but #2818 was reverted by #3106. However @invisiblek fixed it already in https://github.com/tejado/pgoapi/pull/171 5 hours ago. Closing.

**Pull request closed without merging**